### ISSUE TITLE
File decryption error while trying to download a submission

### ISSUE THREAD

**Issue by kushaldas** [freedomofpress/securedrop-workstation#79]
In today's master, if I am trying to download and view any submission. I am getting this error (after I can see sd-gpg starts),  `bad keyword: SUBMISSION_FILE_NO_FILES_FOUND`.

![decrypt_error](https://user-images.githubusercontent.com/272303/38441519-a0517bf8-3a02-11e8-8511-6ede9cf2a00f.png)

**Comment by redshiftzero** [comment]
Thanks for filing this @kushaldas, I ran into this too. Hey @emkll I saw you checked your box in #74 - did you not hit this issue?

**Comment by emkll** [comment]
Yes I can confirm I am experiencing this issue as well, it happens immediately after you authorize the use of the sd-gpg.

**Comment by kushaldas** [comment]
The `decrypt-sd-submission` script can not find any tar.gz files inside of the temporary directory, that is why we are seeing this error. I am working on a PR for this.

**Comment by joshuathayer** [comment]
Hey @kushaldas lmk if you need another eye on this. Does it happen with you download all files (as opposed to a single file)? If not, it might be solved by https://github.com/freedomofpress/securedrop-workstation/pull/78

**Comment by conorsch** [comment]
Just about to run through a test here (on master), will report back if I can reproduce, as well.

**Comment by redshiftzero** [comment]
Unfortunately I hit this bug while processing a download containing multiple files (I did the select all and download flow on the Journalist Interface index page).

**Comment by conorsch** [comment]
So far I'm unable to reproduce the exact error message reported here. I also haven't confirmed a working submission workflow on Qubes 4 stable, so there's definitely still work to be done. One issue I encountered several times was being prompted interactively to choose the target VM for the decryption process—that's definitely new since switching to Qubes 4 stable. Will open separate ticket if I can reproduce it.

**Comment by kushaldas** [comment]
I wrote the code, and then figured out that #78 will fix this. This is the same problem :)

**Comment by kushaldas** [comment]
>Unfortunately I hit this bug while processing a download containing multiple files (I did the select all and download flow on the Journalist Interface index page).

Yes, in case those multiple files has each only one message, we see the same error.

**Comment by kushaldas** [comment]
The PR #78 solves this issue. I can download single or many submissions together, I could not reproduce this error from that branch. @redshiftzero try that PR once.

**Comment by msheiny** [comment]
> One issue I encountered several times was being prompted interactively to choose the target VM for the decryption process—that's definitely new since switching to Qubes 4 stable. Will open separate ticket if I can reproduce it.

@conorsch : fyi @eloquence opened #86 to track this issue. I also hit that
